src/Storages/MergeTree/MergeTreeDataPartDistinctPartitionExpressionCloner.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "MergeTreeDataPartCloner.h"
+
+namespace DB
+{
+    class MergeTreeTransaction;
```

Comment:
Tabulations inside namespaces are not usually used in other code base (and this is not only here but in other new files too, including `.cpp`)